pallets/delegation/src/lib.rs

```diff
@@ -194,16 +191,16 @@ pub mod pallet {
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		#[cfg(feature = "try-runtime")]
 		fn pre_upgrade() -> Result<(), &'static str> {
-			migrations::DelegationStorageMigrator::<T>::pre_migrate()
+			Ok(())
 		}
 
 		fn on_runtime_upgrade() -> Weight {
-			migrations::DelegationStorageMigrator::<T>::migrate()
+			Weight::zero()
 		}
 
 		#[cfg(feature = "try-runtime")]
 		fn post_upgrade() -> Result<(), &'static str> {
-			migrations::DelegationStorageMigrator::<T>::post_migrate()
+			Ok(())
 		}
 	}
```

Comment:
if there are no migrations we should remove the hooks. Otherwise we get the log that migrations where declared in the runtime upgrade log.

Reply:
43be26e